MANIFEST.in

```diff
@@ -6,6 +6,7 @@ include README.*
 include LICENSE
 include CHANGES
 include setup.*
```

Comment:
Should this just say `include setup.py` now?

Reply:
I tried that, and it still included the pyc. I didn't go too far down that rabbit hole.

Reply:
There's also setup.cfg